soh/soh/SohMenuBar.cpp

```diff
@@ -1553,8 +1553,8 @@ void DrawEnhancementsMenu() {
             UIWidgets::Tooltip("Restore a bug from NTSC 1.0 that allows bypassing Bongo Bongo's intro cutscene to quickly kill him");
             UIWidgets::PaddedEnhancementCheckbox("Original RBA Values", CVAR_ENHANCEMENT("RestoreRBAValues"), true, false);
             UIWidgets::Tooltip("Restores the original outcomes when performing Reverse Bottle Adventure.");
-            UIWidgets::PaddedEnhancementCheckbox("Early Eyeball Frog", CVAR_ENHANCEMENT("EarlyEyeballFrog"), true, false);
-            UIWidgets::Tooltip("Restores a bug from NTSC 1.0/1.1 that allows you to obtain the eyeball frog from King Zora instead of the Zora Tunic by holding shield.");
+            UIWidgets::PaddedEnhancementCheckbox("Early Eyeball Frog", CVAR_ENHANCEMENT("EarlyEyeballFrog"), true, false, IS_RANDO, 
+                                                 "This glitch is non-functional in randomiser due to changes in how the items are given.");
```

Comment:
If we're going to have this forced off and untoggleable in rando, we need to have applicable IS_RANDO checks anywhere the CVar is checked in code to make sure the code paths evaluate properly.

Reply:
This is an awkward situation where I didn't choose to disable the setting, but I noticed that it simply wasn't working or worth the mess to fix so disabled the setting for expectations reasons.

Ironically, if I had to spread a ton of extra checks around code that is already questionably invasive, I would probably end up making the mess needed to implement it instead as you've removed the only clean solution!

Reply:
LOL, I'm just saying, only disabling the checkbox during rando isn't really a solution since people could just set it before loading a seed and it would still be enabled after entering a rando save.

Reply:
I am aware, it would just do nothing in that case because the hook execution breaks the glitch.